tools/bench-libs/src/main/java/org/enso/benchmarks/libs/LibBenchRunner.java

```diff
@@ -52,30 +45,45 @@ public static void main(String[] args) {
       }
       System.exit(0);
     }
-    if (cmdOpts.shouldList()) {
-      // TODO: List benchmarks
-      System.exit(0);
-    }
-    List<String> includes = cmdOpts.getIncludes();
-
 
-    File benchRootDir = Paths.get("../../distribution/component/test/Benchmarks").toFile();
-
-    initCtx();
-    var specCollector = new SpecCollector(ctx, rootDir);
-    specCollector.collectBenchSpecsFromSingleFile()
-    Set<BenchSuite> benchSpecs = collectAllBenchSpecs();
-    for (BenchSuite benchSpec : benchSpecs) {
-      for (BenchGroup group : benchSpec.groups()) {
-        System.out.println("Discovered group: " + group.name());
+    File ensoRootDir = Paths.get(System.getProperty("user.dir")).toFile();
```

Comment:
Wouldn't it be better to use `LibBenchRunner.class.getProtectionDomain().getCodeSource().getLocation()` to obtain the location of the class itself and search from it?

Reply:
Sure. I didn't know that this was an option. Thanks for the suggestion.